src/System.Net.HttpListener/src/System/Net/Windows/HttpListener.Windows.cs

```diff
@@ -680,21 +520,7 @@ private void Dispose()
             }
         }
 
-        private uint InternalAddPrefix(string uriPrefix)
-        {
-            uint statusCode = 0;
-
-            statusCode =
-                Interop.HttpApi.HttpAddUrlToUrlGroup(
-                    _urlGroupId,
-                    uriPrefix,
-                    0,
-                    0);
-
-            return statusCode;
-        }
-
-        private bool InternalRemovePrefix(string uriPrefix)
+        private bool RemovePrefixCore(string uriPrefix)
```

Comment:
RemovePrefixCore returns a bool in this impl but void in the other one, and no callers seem to pay attention to this bool.